src/content/en/fundamentals/getting-started/primers/async-functions.md

```diff
@@ -0,0 +1,345 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Async functions allow you to write promise-based code as if it were synchronous
+
+{# wf_published_on: 2016-10-07 #}
+{# wf_updated_on: 2016-10-07 #}
+
+# Async functions - making promises friendly {: .page-title }
+
+{% include "web/_shared/contributors/jakearchibald.html" %}
+
+Async functions are enabled by default in Chrome 55 and they're quite frankly
+marvelous. They allow you to write promise-based code as if it were synchronous,
+but without blocking the main thread. They make your asynchronous code less
+"clever" and more readable.
+
+Async functions look like this:
+
+    async function myFirstAsyncFunction() {
+      try {
+        const fulfilledValue = await promise;
+      }
+      catch (rejectedValue) {
+        // …
+      }
```

Comment:
I see a lot of code samples like this, with the try/catch. I worry that presenting async functions like this may make people think they always need a try/catch block inside their async function body, which definitely isn't true. Maybe the initial example should be a chain of awaiting multiple things, or even a loop, and then explain the exceptions/rejection duality later.